lib/install/validate/network.go

```diff
@@ -444,8 +444,20 @@ func (v *Validator) checkBridgeIPRange(bridgeIPRange *net.IPNet) error {
 }
 
 // getNetwork gets a moref based on the network name
+// distributed port group / logical switch will be used
+// if there are same network name between standard network and distributed port group
 func (v *Validator) getNetwork(op trace.Operation, name string) (object.NetworkReference, error) {
 	defer trace.End(trace.Begin(name, op))
+	// Check format /dc1/network/xxx for standard network usage
+	subnames := strings.SplitN(name, "/", 4)
+	if len(subnames) > 3 {
+		datacenter := subnames[1]
+		if datacenter == v.session.Datacenter.Name() && strings.ToLower(subnames[2]) == "network" {
```

Comment:
Are these checks consistent with other places that we accept inventory paths? I don't recall needing to re-up specify datacenter, but I don't often use systems with multiple datacenters.

Reply:
No, I do not see how other places to check the inventory path, but I can confirm.

Reply:
And this piece code should address the case when the customer specify the datacenter, I think.

Reply:
@zjs I checked the code again, I found the govomomi could support the inventory path by default, so we do not need to address the inventory path by ourselves. Here we can input name or inventory path both like compute-resource parameter.